spec/requests/api/v1/topics/show_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+describe "POST api/v1/topics", type: :request do
```

Comment:
Maybe we can change this,  is not a POST
And
This file should be named `index_spec.rb` right?